bundles/CoreBundle/Command/ThumbnailsImageCommand.php

```diff
@@ -65,13 +75,29 @@ protected function configure()
                 null,
                 InputOption::VALUE_NONE,
                 'do not generate high-res (@2x) versions of thumbnails'
+            )->addOption(
+                'processes',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'The number of parallel processes to run',
+                1
+            )->addOption(
+                'child',
+                null,
+                InputOption::VALUE_NONE,
+                'Set on child processes'
+            )->addArgument(
+                'item',
+                InputArgument::OPTIONAL,
+                'The item to process'
```

Comment:
We should make clear here that this is just used internally, nothing to care about 😉

Reply:
done!